common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -331,6 +363,17 @@ bool base_wlan_hal_dwpal::fsm_setup()
 
             m_fd_ext_events = -1;
 
+            // detach nl socket from main vap
+            if (!m_dwpal_nl_ctx) {
+                LOG(DEBUG) << "detaching nl interface";
+                if (dwpal_driver_nl_detach(&m_dwpal_nl_ctx) == DWPAL_FAILURE) {
+                    LOG(DEBUG) << "dwpal_driver_nl_detach() success for radio ="
```

Comment:
Error message is incorrect. If we got `DWPAL_FAILURE`, it's most definitely not a success...

Reply:
done